client/src/components/App.jsx

```diff
@@ -1,15 +1,16 @@
-import Review from './components/Review.jsx';
-import Price from './components/Price.jsx';
-import Selection from './components/Selection.jsx';
-import Distribution from './components/Distribution.jsx';
-import Highlight from './components/Highlight.jsx';
+import React from 'react';
 import $ from 'jquery';
-import './style.css';
```

Comment:
Did you take this out because you're now going to use grunt or gulp?

Reply:
I moved it to index.jsx